cms/static/js/views/pages/container.js

```diff
@@ -18,7 +18,8 @@ define(["jquery", "underscore", "gettext", "js/views/pages/base_page", "js/views
                 "click .visibility-button": "editVisibilitySettings",
                 "click .duplicate-button": "duplicateXBlock",
                 "click .delete-button": "deleteXBlock",
-                "click .new-component-button": "scrollToNewComponentButtons"
+                "click .new-component-button": "scrollToNewComponentButtons",
+                "click .toggle-preview-button": "toggleChildrenPreviews"
```

Comment:
Should a default, no-op implementation of toggleChildrenPreviews exist in this class?